.github/assets/testing/edge.yml

```diff
@@ -50,6 +50,15 @@ core:
         storage:
           database: 2G
 features:
+  baremetal:
+    software:
+      charms:
+        ironic-conductor-k8s:
+          channel: 2025.1/beta
```

Comment:
Can you make all channels from beta to edge.
Later once 2024.1 charms are published, this need to be changed to 2024.1 as well